anda/kotlin/kotlin-native/kotlin-native.spec

```diff
@@ -33,6 +33,10 @@ sed -i "s|\(DIR *= *\).*|\1%{_bindir}|" bin/*
 sed -i "s|\(KONAN_HOME *= *\).*|\1%{_datadir}/%{name}|" bin/*
 
 
+%build
+true
```

Comment:
leave it empty?

Reply:
you can just leave it empty. [Check this out](https://github.com/terrapkg/packages/blob/main/anda/fonts/seto/seto-fonts.spec)